couchbase-entity/src/main/java/com/kaufland/generation/model/EntityGeneration.kt

```diff
@@ -30,9 +30,10 @@ class EntityGeneration {
         companionSpec.addFunctions(create(holder, useSuspend))
         companionSpec.addFunction(findById(holder, useSuspend))
         companionSpec.addFunction(findByIds(holder, useSuspend))
+        companionSpec.addFunctions(TypeConversionMethodsGeneration(useSuspend).generate())
```

Comment:
moved from the spec of the instance to the companion-object because it does not need the instance-fields but is now needed in findBy...() methods